Userland/Libraries/LibWeb/FileAPI/Blob.cpp

```diff
@@ -12,8 +12,56 @@
 
 namespace Web::FileAPI {
 
+// https://w3c.github.io/FileAPI/#convert-line-endings-to-native
+static ErrorOr<String> convert_line_endings_to_native(String const& string)
+{
+    // 1. Let native line ending be be the code point U+000A LF.
+    StringView native_line_ending = "\n"sv;
+
+    // FIXME: 2. If the underlying platform’s conventions are to represent newlines as a carriage return and line feed sequence, set native line ending to the code point U+000D CR followed by the code point U+000A LF.
```

Comment:
Why is this FIXME'd?

Reply:
The only platform I could think of that uses `\r\n` is Windows and we do not yet support that. Also we have no runtime mechanism (or preprocessor flag for Windows) to detect if the underlying platform’s conventions are to represent newlines as a carriage return and line feed sequence.

Reply:
LibWeb doesn't even compile on Windows, whoever ends up porting it can take care of this. FIXME is for us regular developers, but there is no action here :)

Reply:
so, let's just remove the fixme part and leave it as a no-op comment.

Reply:
Ah, should I remove the FIXME and let add a note about it being a no-op?

Reply:
I'll do that then :)

Reply:
Done

Reply:
Or wait a second...

Reply:
Now it's done